Classes/Service/ImageResizer.php

```diff
@@ -308,11 +308,45 @@ public function processFile(
         $isRotated = false;
 
         if ((bool)$ruleset['auto_orient'] === true) {
-            $orientation = ImageUtility::getOrientation($fileName);
+            $orientation = ImageUtility::getOrientation($fileName);			
```

Comment:
did you add useless white spaces here? why? (on my smartphone so maybe not seeing correctly)